shaketune/dummy_macros.cfg

```diff
@@ -45,8 +45,16 @@ gcode:
 [gcode_macro COMPARE_BELTS_RESPONSES]
 description: dummy
 gcode:
-    {% set freq_start = params.FREQ_START|default(5) %}
-    {% set freq_end = params.FREQ_END|default(133.33) %}
+    {% if params.FREQ_START is defined %}
+        {% set freq_start = "FREQ_START=" ~ params.FREQ_START|default(5) %}
+    {% else %}
+        {% set freq_start = "" %}
+    {% endif %}
+    {% if params.FREQ_END is defined %}
+        {% set freq_end = "FREQ_END=" ~ params.FREQ_END|default(133.33) %}
+    {% else %}
+        {% set freq_end = "" %}
+    {% endif %}
```

Comment:
Actually this will not work. I would prefer to use the same pattern as the other variables that are not mandatory like accel_per_hz on this block. So not have a default(xxx) and fill it to '' in the params_filtered object.

Reply:
Fair enough, I'll rework it later